packages/relay-kit/src/packs/safe-4337/types.ts

```diff
@@ -156,6 +158,7 @@ export type EstimateFeeFunction = ({
 export interface IFeeEstimator {
   preEstimateUserOperationGas?: EstimateFeeFunction
   postEstimateUserOperationGas?: EstimateFeeFunction
+  defaultVerificationGasLimitOverhead?: bigint
```

Comment:
This could be as well a constructor parameter for the estimator right? Instead adding it to the main `Safe4337Pack` estimation method

Reply:
I can add the defaultVerificationGasLimitOverhead logic to postEstimateUserOperationGas, but
i would need to add threshold to EstimateFeeFunctionProps.

Reply:
done